scripts/skaffold.install.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64
+chmod +x skaffold
+mv skaffold ${HOME}
```

Comment:
Not sure it's good idea to move something to ${HOME}. Not a big problem on travis-ci machine, but kind of unexpected on local one.

Reply:
Fixed